cmd/exec/main.go

```diff
@@ -49,12 +48,10 @@ func main() {
 	rootCmd.AddCommand(execCmd)
 	rootCmd.AddCommand(pushCmd)
 	rootCmd.AddCommand(setCmd)
-	rootCmd.AddCommand(eventCmd)
-	rootCmd.AddCommand(eventfilterCmd)
-	eventfilterCmd.AddCommand(createCmd)
-	eventfilterCmd.AddCommand(deleteCmd)
-	eventfilterCmd.AddCommand(updateCmd)
-	eventfilterCmd.AddCommand(applyCmd)
+	rootCmd.AddCommand(eventsCmd)
```

Comment:
I've moved some commands around and merged other. Run `exec events` to see what ones exist.